sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/implementation/KeyVerifyRequest.java

```diff
@@ -17,7 +17,7 @@ public final class KeyVerifyRequest {
      * The signing/verification algorithm. For more information on possible
      * algorithm types, see SignatureAlgorithm. Possible values
      * include: 'PS256', 'PS384', 'PS512', 'RS256', 'RS384', 'RS512', 'RSNULL',
-     * 'ES256', 'ES384', 'ES512', 'ES256K'.
+     * 'ES256', 'ES384', 'ES512', 'ES256K', 'Ed25519'.
```

Comment:
Shouldn't this be `EdDSA`?

Reply:
No, the services specifies `Ed25519` as the curve name [here](https://github.com/Azure/azure-rest-api-specs/blob/21826324d0c76ef0309fe7761eaa34d4da174359/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json#L1386).

Reply:
That is for curve this one signature algorithm - [https://github.com/Azure/azure-rest-api-specs/blob/21826324d0c76ef0309fe7761eaa34d4da174359/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json#L1901](https://github.com/Azure/azure-rest-api-specs/blob/21826324d0c76ef0309fe7761eaa34d4da174359/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json#L1901)

Reply:
That one is for curve Eddsa would be for signature algorithm - [https://github.com/Azure/azure-rest-api-specs/blob/21826324d0c76ef0309fe7761eaa34d4da174359/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json#L1983](https://github.com/Azure/azure-rest-api-specs/blob/21826324d0c76ef0309fe7761eaa34d4da174359/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json#L1983)

Reply:
You're both right, my bad! :P